style.css

```diff
@@ -70,13 +83,14 @@ header ul {
 /* ----- MAIN SECTION ----- */
 
 #main {
-  display: flex;
-  justify-content: center;
+  background-color: var(--html-test-background);
 }
 
 .main {
-  max-width: 1000px;
-  max-height: 120vh;
+  width: 100vw;
```

Comment:
- [ ] **[OPTIONAL]** Great job with your project, I can see that you've put a lot of effort into it! I just wanted to **offer a small suggestion that could help make it look even better**. I noticed that s**ome of your sections are going outside of the width of the page**, and this might be because you're using `100vw` for their `width`. To fix this, you could try changing the `width` to `100%` instead. 🙌

  **_I've included a quick video to show you what I mean and how you can make this change.🎬_**
  
  [5_12_2023_12-21-42 AM.webm](https://github.com/rudicarrilloypr/portfolio/assets/107952724/8bc8d123-38c2-4842-96b8-bf2017dd2425)